notebooks/1-sequence-stats.jmd

```diff
@@ -0,0 +1,147 @@
+using Plots: make_fillrange_from_ribbon
+# Coronavirus Analysis
+Working directory must be Cov2_Analysis directory
+
+## Mean and Standard Deviation
+Finding mean sequence length, standard devation for length, mean gc content and standard deviation for gc content for CoV sequences.
+
+```julia 
+using BISC195Bioinformatics
+
+return mean_and_std_fasta("data/CoV_sequences.fasta")
+```
+
+## Minimum and Maximum
+
+Finding length of smallest and largest sequence for CoV sequences. 
+
+```julia
+seq_lengths = sequence_lengths("data/CoV_sequences.fasta")
+
+min_length = minimum(seq_lengths)
+max_length = maximum(seq_lengths)
+
+return min_length, max_length
+```
+
+## Identifying Outliers
+
+Using histogram to spot sequence length outliers in CoV sequences. Added more descriptive axis labels and removed legend.
+
+```julia
+using Plots
+
+histogram(seq_lengths, 
+xlabel = "Sequence Length",
```

Comment:
Typically would indent these lines to make it clear at a glance that they're part of the call to `histogram`